kubernetes/cronjob.yaml

```
@@ -0,0 +1,47 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  labels:
+    app.kubernetes.io/name: vault-snapshot
+    app.kubernetes.io/version: v0.1.0
+  name: vault-snapshot
+spec:
+  jobTemplate:
+    schedule: 0 4 * * *
+    metadata:
+      labels:
+        app.kubernetes.io/name: vault-snapshot
+        app.kubernetes.io/version: v0.1.0
+    spec:
+      template:
+        metadata:
+          labels:
+            app.kubernetes.io/name: vault-snapshot
+            app.kubernetes.io/version: v0.1.0
+        spec:
```

As this might be executed in very controlled environment we should use a dedicated Service account and be explicit about mounting the Service account token:
```suggestion
        spec:
          automountServiceAccountToken: true
          serviceAccountName: vault-raft-backup
```